script/setup/addNewProject.ts

```diff
@@ -461,16 +514,18 @@ export const buildProjectConstants = async (
   };
 
   for (const token of tokenInfo.tokens) {
+    const poolCount = tokensWithOnePoolId.includes(token) ? 1 : 0;
     const limitsAndPoolId = {};
     for (const chain of allChains) {
       limitsAndPoolId[chain] = {
-        [IntegrationTypes.fast]: tokenLimitInfo[token],
+        [IntegrationTypes.fast]: { ...tokenLimitInfo[token], poolCount },
```

Comment:
Can we avoid passing poolCount when it is zero? so that project constants file stays clean?